.buildkite/commands/run-ui-tests.sh

```diff
@@ -36,7 +36,7 @@ echo "--- 🔬 Testing"
 xcrun simctl list >> /dev/null
 rake mocks &
 set +e
-bundle exec fastlane test_without_building name:"$TEST_NAME" try_count:3 device:"$DEVICE" ios_version:"$IOS_VERSION"
+bundle exec fastlane test_without_building name:WordPressUITests device:"$DEVICE" ios_version:"$IOS_VERSION"
```

Comment:
Regarding `try_count: 3`: That parameter is not part of Fastlane `run_test` (`scan`). It was a feature provided by a plugin we long since removed, see:

- https://github.com/wordpress-mobile/WordPress-iOS/commit/75f87334ee967e1ae8c00d8ccfbecef28fef237d
- https://github.com/wordpress-mobile/WordPress-iOS/pull/17456
- https://github.com/wordpress-mobile/WordPress-iOS/pull/17841

The parameter was removed from the CI script at the time, but reappeared when moving from CircleCI to Buildkite.

Reply:
It's good to have it removed, because `WordPressUITests` test plan in Xcode also has `Retry on Failure` set to 3, so now it's potentially less confusing for a person that will try to adjust tests repetitions.